### ISSUE TITLE
Feature Request: Config high and low

### ISSUE THREAD

**Issue by paleoCoder** [Rytiggy/api-watchface#12]
Add a config for the high and low value graph lines.

**Comment by Rytiggy** [comment]
I like this

**Comment by danpowell88** [comment]
Ideally this should be pulled from nightscout or the xdrip apis

**Comment by PedanticAvenger** [comment]
Is this even available from one of the api endpoints? Best to pull it from xdrip for user experience consistency but suspect we may have to make it a watch face setting.

**Comment by paleoCoder** [comment]
I don't use xdrip on my phone. I configured the watchface to go off of my
Nightscout site and it works great. I have Nightscout setup with a bridge
into Dexcom to pull all the data into the Nightscout DB.

**Comment by PedanticAvenger** [comment]
Fair enough, this is why I've asked for the units hint to be integrated into Nightscout as well as Xdrip, but the question remains the same wether XDrip or Nightscout.  does any of the API endpoints provide the configured high/low thresholds?
Hmm, also need to integrate watch vibe for these as well now that I think about it.
:)

**Comment by paleoCoder** [comment]
Nightscout has the data here: http://localhost:1337/api/v1/status.json
Documentation: https://github.com/nightscout/cgm-remote-monitor#nightscout-api

Sample JSON:
```
"thresholds": {
      "bgHigh": 160,
      "bgTargetTop": 120,
      "bgTargetBottom": 70,
      "bgLow": 60
    },

```

**Comment by danpowell88** [comment]
I'm sure Jamorham would add it to the xdrip feed if it wasnt already there

**Comment by PedanticAvenger** [comment]
I agree, I'll ask him to look here for a comment.

**Comment by jamorham** [comment]
I can emulate the `bgHigh` and `bgLow` elements of the `status.json` endpoint.
I can also make an option to reduce the amount of data fields coming back from the `sgv.json` endpoint.

**Comment by jamorham** [comment]
bare bones `status.json` added in the Feb 9th Nightly. Changes here: https://github.com/NightscoutFoundation/xDrip/commit/afaa84d0740979464c6017acf7ea47fd9abfbeba instructions in the usual local services document.

Also a `brief_mode` option for `sgv.json` to reduce the data overhead. If you guys tell me which date field you are using I could trim it further. It reduces it to about 60% so far I think. @swissalpine

**Comment by swissalpine** [comment]
That's awesome!
For (my) Garmin Apps I don't need the strings "dateString" and "sysTime".
@jamorham

**Comment by jamorham** [comment]
@Rytiggy Which field do you use for date of sgv record? For garmin only the `date` field is being used. If the same is true for your watchface then I can trim some more of the date fields and reduce the data overhead even further.

**Comment by Rytiggy** [comment]
I can make if work with just the date field.

**Comment by PedanticAvenger** [comment]
Should units_hint be perhaps moved from sgv.json to status.json?  Feels most like it belongs there as it's a system config item rather than a data value per se.
@jamorham @swissalpine @Rytiggy

**Comment by swissalpine** [comment]
For me it's good as it is. Garmin needs some seconds to proceed a response.
Firing two requests retards the first show of BG reading.


Am 11.02.2018 10:10 nachm. schrieb "raymond-richmond" <
notifications@github.com>:
